modules/ROOT/pages/legacy-search-experience.adoc

```diff
@@ -0,0 +1,312 @@
+= Legacy Search Experience
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+The legacy search experience, which was previously called the old search, is available in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. This search experience does not include the enhanced options for searching by rank, organizations, and lifecycle states. The enhanced search is the default search experience when logging into Exchange.
+
+To access the legacy search, click the *Back to legacy search* button from the landing page.
+
+== Display Assets in List or Grid View
+
+You have an option to display the catalog of assets in list or grid view. List view arranges assets in a list format and grid view displays the assets in tiles. Both views display the asset name, type, rating, and created by information.
+
+== Filter Search Results
+
+You can filter search results by asset types (for example, API groups, REST APIs, and rulesets) by selecting an option from the  *All types* drop-down menu. The options that appear depend on which types are available within an organization.
+ 
+ 
+== Search in Organization
+
+You can search only in the Exchange organization that you are currently viewing. If you are viewing a private Exchange organization, the search lists only that organization’s assets and not MuleSoft public assets.
+
+== Save a Search
+
+You can save search criteria that you use often for quick discovery of specific assets.
+
+. Enter a search term and click *Save This Search*.
+. When prompted, give the search a name and select whether the search is an organization search or personal search.
++
+Administrators can save searches visible to anyone in their current business group. Personal searches are visible only to the person who creates them.
+. Save the search.
+. Confirm that your search appears in the saved searches in the left navigation bar.
++
+Exchange lists search terms specific to your business group and the saved searches you created:
++
+image::ex2-saved-searches-groups.png[Saved Search Groups]
+
+== Search API Specifications
+
+You can use the following API specification properties are used in search:
```

Comment:
I think you meant: "You can use the following API specification properties that are used in search:".

Reply:
will fix